CONTRIBUTING.md

```diff
@@ -4,17 +4,32 @@ Thanks for your interest in contributing to Turbo!
 
 **Important note**: At the moment, Turbo is made up of two tools, Turborepo and Turbopack, built with different languages and toolchains. In the future, Turbo will become a single toolchain built on Rust and the Turbo engine. In the meantime, please follow the respective guide when contributing to each tool:
 
-- [Contributing to Turborepo](#contributing-to-turborepo)
-- [Contributing to Turbopack](#contributing-to-turbopack)
+- [Contributing to Turbo](#contributing-to-turbo)
+  - [Contributing to Turborepo](#contributing-to-turborepo)
+    - [Building Turborepo](#building-turborepo)
+    - [Testing Turborepo](#testing-turborepo)
+  - [Debugging Turborepo](#debugging-turborepo)
+  - [Benchmarking Turborepo](#benchmarking-turborepo)
+  - [Updating `turbo`](#updating-turbo)
+  - [Publishing `turbo` to the npm registry](#publishing-turbo-to-the-npm-registry)
+  - [Contributing to Turbopack](#contributing-to-turbopack)
+    - [Testing Turbopack](#testing-turbopack)
+    - [Benchmarking Turbopack](#benchmarking-turbopack)
+  - [Troubleshooting](#troubleshooting)
 
 ## Contributing to Turborepo
 
 ### Building Turborepo
 
 Dependencies
 
-1.  On OSX: `brew install sponge jq protobuf protoc-gen-go protoc-gen-go-grpc golang`
-1.  Run `pnpm install` at root
+1. Install `jq` and `sponge`
```

Comment:
I removed the other dependencies here because those are installed in the below step